api/python/provisioner/commands/__init__.py

```diff
@@ -431,6 +438,34 @@ def run(self, *args, targets=ALL_MINIONS, dry_run=False, **kwargs):
 
         return self._run(params, targets)
 
+@attr.s(auto_attribs=True)
+class PillarDump(CommandParserFillerMixin):
+    input_type: Type[inputs.PillarKeysList] = inputs.PillarKeysList
+    _run_args_type = RunArgsPillarDump
+
+    @classmethod
+    def from_spec(
+        cls, input_type: str = 'PillarKeysList'
+    ):
+        return cls(input_type=getattr(inputs, input_type))
+
+
+    def run(
+        self, *args, targets: str = ALL_MINIONS, local: bool = False,
+        **kwargs
+    ):
+        res = PillarGet.run(self)
+        os.makedirs(f"{PRVSNR_CORTX_CONFIG_DIR}", exist_ok=True)
+
+        resp_loaded = json.loads(json.dumps(res).replace('null', '""'))
```

Comment:
res is dict()
don't make any operations here. This line is not required.

Reply:
@ypise @andkononykhin @nitin-seagate 

This is how the converted data would looks like (a sample block) 

`
                "cluster_id": "a918a173-c8e4-4231-ac11-d3472d265291",
                "hostname": "ssc-vm-c-1210.colo.seagate.com",
                "is_primary": **"True"**,
                "network": {
                    "data": {
                        "public_ip_addr": "172.20.30.101"
                    },
                    "data_nw": {
                        "gateway": **"None"**,
                        "iface": [
                            "eth1",
                            "eth2"
                        ],
                        "netmask": "255.255.0.0",
                        "public_ip_addr": "None",
                        "pvt_ip_addr": "",
                        "roaming_ip": "127.0.0.1"
                    }
`